scripts/monitoring/cron-send-ssl-check.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+""" check ssl cert info"""
+
+# We just want to see any exception that happens
+# don't want the script to die under any cicumstances
+# script must try to clean itself up
+# pylint: disable=broad-except
+
+# main() function has a lot of setup and error handling
+# pylint: disable=too-many-statements
+
+# main() function raises a captured exception if there is one
+# pylint: disable=raising-bad-type
+
+# Adding the ignore because it does not like the naming of the script
+# to be different than the class name
+# pylint: disable=invalid-name
+
+import argparse
+import datetime
+import random
+import string
+import time
+import urllib2
+import ssl
+import socket
+import logging
+import OpenSSL
+
+
+# Our jenkins server does not include these rpms.
+# In the future we might move this to a container where these
+# libs might exist
+#pylint: disable=import-error
+from openshift_tools.monitoring.ocutil import OCUtil
+from openshift_tools.monitoring.metric_sender import MetricSender
+
+
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+ocutil = OCUtil()
+
+commandDelay = 5 # seconds
+# use parsed arg instead
+#testLoopCountMax = 180 # * commandDelay = 15min
+testCurlCountMax = 18 # * commandDelay = 1min30s
+testNoPodCountMax = 18 # * commandDelay = 1min30s
+
+def runOCcmd(cmd, base_cmd='oc'):
+    """ log commands through ocutil """
+    logger.info(base_cmd + " " + cmd)
+    oc_time = time.time()
+    oc_result = ocutil.run_user_cmd(cmd, base_cmd=base_cmd, )
+    logger.info("oc command took %s seconds", str(time.time() - oc_time))
+    return oc_result
+
+def runOCcmd_yaml(cmd, base_cmd='oc'):
+    """ log commands through ocutil """
+    logger.info(base_cmd + " " + cmd)
+    ocy_time = time.time()
+    ocy_result = ocutil.run_user_cmd_yaml(cmd, base_cmd=base_cmd, )
+    logger.info("oc command took %s seconds", str(time.time() - ocy_time))
+    return ocy_result
+
+def parse_args():
+    """ parse the args from the cli """
+    logger.debug("parse_args()")
+
+    parser = argparse.ArgumentParser(description='OpenShift public ssl expired info check ')
+    parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose?')
+    return parser.parse_args()
+
+def send_metrics(day_left):
+    """ send data to MetricSender"""
+    logger.debug("send_metrics()")
+
+    ms_time = time.time()
+    ms = MetricSender()
+    logger.info("Send data to MetricSender")
+    ms.add_metric({'openshift.master.public.ssl.left': day_left})
+    ms.send_metrics()
+    logger.info("Data sent to Zagg in %s seconds", str(time.time() - ms_time))
+
+
+
+def get_ssl_info(domain_name):
```

Comment:
please rename this function `get_ssl_certificate_expiry_days` or something more descriptive